### PR TITLE
Implement analyzer for `salary-calculator` after revert

### PR DESCRIPTION
**Pull request by manumafe98**
closes [#111](https://github.com/exercism/java-analyzer/issues/111)

Todo:
- [x] Add website-copy comments
- [x] Merge https://github.com/exercism/website-copy/pull/2337

### CODE REVIEW

**Comment by manumafe98** [comment]
Just in case, do not merge until the website-copy comments are merged as well!

File: src/main/java/analyzer/exercises/salarycalculator/SalaryCalculatorAnalyzer.java
Comment (sanderploegsma) [inline src/main/java/analyzer/exercises/salarycalculator/SalaryCalculatorAnalyzer.java:42]: I think you can simplify the other `if` statements by adding this to the top of this method:

```java
if (essentialCommentAdded) {
    return;
}
```

Then you don't need to keep checking `!essentialCommentAdded`.

**Review by sanderploegsma — APPROVED** [review]
Nice work! One small suggestion to make the code a bit more readable, feel free to address it if you want. Otherwise, fine to merge as-is! 👍 

... As soon as the copy is merged as well, of course 🙃

File: src/main/java/analyzer/exercises/salarycalculator/SalaryCalculatorAnalyzer.java
Comment (manumafe98) [reply src/main/java/analyzer/exercises/salarycalculator/SalaryCalculatorAnalyzer.java:42]: Oh yeah! thanks for that suggestion! also I updated that from the secrets analyzer because I used a similar logic!

**Review by sanderploegsma — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**
